### ISSUE TITLE
Include version number in README of each crate

### ISSUE THREAD

**Issue by ggwpez** [paritytech/parity-publish#41]
The README of each crate should mention the version number at the end. This makes it possible to reverse-lookup into what release a crate version was included.

**Comment by Morganamilo** [comment]
What's the plan for this? A magic `sed` is the only way I can think to do it but that feels messy and easily breakable if some one changes the readme format unknowingly.

**Comment by ggwpez** [comment]
> What's the plan for this? A magic `sed` is the only way I can think to do it but that feels messy and easily breakable if some one changes the readme format unknowingly.

Hm not sure i get what you mean, so maybe Im oversimplifying it but can we do something like this?
`echo "\nThe corresponding Polkadot-SDK release for this version is: $VERSION" >> README.md`

**Comment by Morganamilo** [comment]
I guess that works if we want it right at the bottom of the readme.

**Comment by Morganamilo** [comment]
I've written it so it appends

```
## Release

{description}
```

where description is something like `Polkadot SDK v1.2.3`.

It's this way because the tool should be repo agnostic so I don't want to hardcode polkadot into the sentence and I generally prefer readme's to be less wordy.

Let me know if this wording works for you.

**Comment by ggwpez** [comment]
Yep sounds good, thanks! Then what do we do with the crate description?  
crates-io only shows the latest description which is then wrong for old versions. Maybe we should remove it to avoid confusion?
